safekeeper/spec/ProposerAcceptorReconfig.tla

```diff
@@ -291,18 +332,17 @@ LogSafety == PAS!LogSafety
 \* Invariants which don't need to hold, but useful for playing/debugging.
 \********************************************************************************
 
+\* Check that we ever switch into non joint conf.
+MaxAccConf == ~ \E a \in acceptors:
+    /\ acc_conf[a].generation = 3
+    /\ acc_conf[a].newMembers /= NULL
```

Comment:
1. "ever", not "never"?
2. Why hard-code the =3? Aren't all odd generations non-joint?

Reply:
1. Well, this formula itself states that acceptor with non joint conf doesn't exist ("never"). But once you put it into invariants the idea is that it gets violated, so it allows to "Check that we *ever* switch into non joint conf.", if that makes sense.
2. Yes, nice observation.